src/applications/gqrx/mainwindow.cpp

```diff
@@ -1540,21 +1541,49 @@ void MainWindow::stopAudioStreaming()
 }
 
 /** Start I/Q recording. */
-void MainWindow::startIqRecording(const QString& recdir)
+void MainWindow::startIqRecording(const QString& recdir, const QString& format)
 {
     qDebug() << __func__;
     // generate file name using date, time, rf freq in kHz and BW in Hz
     // gqrx_iq_yyyymmdd_hhmmss_freq_bw_fc.raw
     auto freq = (qint64)(rx->get_rf_freq());
     auto sr = (qint64)(rx->get_input_rate());
     auto dec = (quint32)(rx->get_input_decim());
-    auto lastRec = QDateTime::currentDateTimeUtc().
-            toString("%1/gqrx_yyyyMMdd_hhmmss_%2_%3_fc.'raw'")
-            .arg(recdir).arg(freq).arg(sr/dec);
+    auto currentDate = QDateTime::currentDateTimeUtc();
+    auto filenameTemplate = currentDate.toString("%1/gqrx_yyyyMMdd_hhmmss_%2_%3_fc.%4").arg(recdir).arg(freq).arg(sr/dec);
+    bool sigmf = (format == "SigMF");
+    auto lastRec = filenameTemplate.arg(sigmf ? "sigmf-data" : "raw");
+
+    QFile metaFile(filenameTemplate.arg("sigmf-meta"));
+    bool ok = true;
+    if (sigmf) {
+        auto meta = QJsonDocument { QJsonObject {
+            {"global", QJsonObject {
+                {"core:datatype", "cf32_le"}, // does anyone use gqrx on big endian platforms?
+                {"core:sample_rate", sr/dec},
+                {"core:version", "v1.0.0"},
```

Comment:
```suggestion
                {"core:version", "1.0.0"},
```
The spec is unclear on this, but I believe the correct version number format is `1.0.0`.

I've proposed a clarification in the standard: https://github.com/sigmf/SigMF/pull/298